services/conda/conda.tester.js

```diff
@@ -87,27 +86,13 @@ t.create('downloads (skip prefix, relabel)')
 
 t.create('unknown package')
   .get('/d/conda-forge/some-bogus-package-that-never-exists.json')
-  .expectJSON({ name: 'conda|downloads', value: 'invalid' })
+  .expectJSON({ name: 'conda', value: 'not found' })
 
 t.create('unknown channel')
   .get('/d/some-bogus-channel-that-never-exists/zlib.json')
-  .expectJSON({ name: 'conda|downloads', value: 'invalid' })
+  .expectJSON({ name: 'conda', value: 'not found' })
```

Comment:
We use the label from `defaultBadgeData()` here now. As with some other recent badges, I think we can live with losing the customisation in the error case :man_shrugging: